rfcs/button-component.md

```diff
@@ -80,7 +80,7 @@ Rough DOM composition:
   * `EndEnhancer: ReactComponent` - Optional
 * `size: 'default' | 'compact'`
   Defines the size of a button.
-* `shape: 'round' | 'square'`
+* `shape: 'default' | 'round' | 'square'`
```

Comment:
Having an additional `default` shape allows us to cater specific styling to the truly `square` shape:

![image](https://user-images.githubusercontent.com/2263443/45202343-89cd3680-b22d-11e8-9b8f-7478105eb962.png)